src/Engine/Renderer/Camera/Camera.hpp

```diff
@@ -6,34 +6,29 @@
 #include <Core/Math/LinearAlgebra.hpp>
 #include <Core/Math/Ray.hpp>
 
+#include <Engine/Component/Component.hpp>
+
 namespace Ra {
 namespace Engine {
-// The class Camera defines a camera in 3D space with
-class RA_ENGINE_API Camera final {
+
+// FIXME (Hugo) To me this class could totally be renamed LightComponent and get a Light struct
+// embedded. Thoughts are welcome !
+/// A Light is an Engine Component storing a light object.

```

Comment:
What is the meaning of this two comment? we are speaking about cameras here, not lights